app/db/models.py

```diff
@@ -214,6 +214,7 @@ class ProxyHost(Base):
     is_disabled = Column(Boolean, nullable=True, default=False)
     mux_enable = Column(Boolean, nullable=False, default=False, server_default='0')
     fragment_setting = Column(String(100), nullable=True)
+    noise_setting = Column(String(100), nullable=True)
```

Comment:
noise its only for wireguard and currently its not supported

Reply:
No it work with splithttp h3 too